src/content/tools/devtools/network.md

```diff
@@ -9,20 +9,37 @@ The network view works with all Flutter and Dart applications.
 
 ## What is it?
 
-The network view allows you to inspect HTTP, HTTPS, and web socket traffic from
+The network view allows you to inspect HTTP, HTTPS, and WebSocket traffic from
 your Dart or Flutter application.
 
 ![Screenshot of the network screen](/assets/images/docs/tools/devtools/network_screenshot.png){:width="100%"}
 
+## What network traffic is recorded?
+
+All network traffic that originates from `dart:io` (like the
+[`HttpClient`][HttpClient] class) or that is logged using the
+[`http_profile`][http_profile] package is recorded in the network request
+table. This includes network traffic from the
+[`cupertino_http`][cupertino_http], [`cronet_http`][cronet_http], and
```

Comment:
worth mentioning dio here too? This seems to be a fairly popular networking package

Reply:
I feel a little reluctant to promise that certain third party packages do this or that. Even in this case, using dio on a web app will _not_ be recorded, as we don't log anything from `dart:html` or JS interop. But I could add something if it really seems to be the premiere network request app.

Reply:
Oh okay. Maybe I'm misunderstanding what the listed packages are for. I was under the impression that some of these were 3P packages we were listing.

Reply:
Well, maybe it's happenstance, but these are the packages that support logging via the http_profile package. They are all 1p (I think ok_http was an intern project or something, but still Google?). I guess since we have integration tests for dio, we can list it here, haha.